src/components/molecules/TimePickerInput/TimePickerInput.tsx

```diff
@@ -0,0 +1,161 @@
+import {
+  useState,
+  useEffect,
+  FC,
+  forwardRef,
+  Ref,
+  ForwardedRef,
+  useRef,
+} from 'react'
+import TimeColumn from 'components/atoms/TimeColumn/TimeColumn'
+import { ReactComponent as Clock } from 'assets/icons/clock.svg'
+import { FieldError } from 'react-hook-form'
+import Icon from 'components/atoms/Icon/Icon'
+import classNames from 'classnames'
+
+import classes from './styles.module.scss'
+
+export type TimePickerInputProps = {
```

Comment:
**[blocker]**

Let's remove some type repetition from here as well. It can be quite confusing, since it's hard to tell what props have the same type between parent and child (which ones are passed directly from parent to child) and which ones are new/modified